apps/website/src/components/variables/_item.astro

```diff
@@ -11,7 +11,7 @@
     display: flex;
     align-items: center;
     justify-content: center;
-    background-color: var(--iui-color-background-1);
+    background-color: var(--iui-color-background);
```

Comment:
Maybe undo this and the other changes in website (`variables.mdx`) since most of it is still broken.

Reply:
Undid those changes in variables.mdx